src/ixdat/techniques/ec.py

```diff
@@ -1,18 +1,597 @@
+    - `current`: The working-electrode current typically in [mA] or [mA/cm^2].
+        `ec_meas["current"]` always returns a `ValueSeries` characterized by:
+        - normalized if the measurement has been normalized with the electrode area
+            (`A_el`, see `normalize`)
+        - A name that makes clear whether it is normalized
+        - Data which spans the entire timespan of the measurement
+    - `selector`: A counter series distinguishing sections of the measurement program.
+        This is essential for analysis of complex measurements as it allows for
+        corresponding parts of experiments to be isolated and treated identically.
+        `selector` in `ECMeasurement` is defined to incriment each time one or more of
+        the following changes:
+        - `loop_number`: A parameter saved by some potentiostats (e.g. BioLogic) which
+            allow complex looped electrochemistry programs.
+        - `file_number`: The id of the component measurement from which each section of
+            the data (the origin of each `ValueSeries` concatenated to `potential`)
+        - `cycle_number`: An incrementer within a file saved by a potentiostat.
+
+    The names of these ValueSeries, which can also be used to index the measurement, are
+    conveniently available as properties:
+    - `ec_meas.t_str` is the name of the definitive time, which corresponds to potential.
+    - `ec_meas.E_str` is the name of the raw potential
+    - `ec_meas.V_str` is the name to the calibrated and/or corrected potential
+    - `ec_meas.I_str` is the name of the raw current
+    - `ec_meas.J_str` is the name of the normalized current.
+    - `ec_meas.sel_str` is the name of the default selector, i.e. "selector"
+
+    Numpy arrays from important `DataSeries` are also directly accessible via attributes:
+    - `ec_meas.t` for `ec_meas["potential"].t`
+    - `ec_meas.v` for `ec_meas["potential"].data`
+    - `ec_meas.j` for `ec_meas["current"].data
 
-    extra_column_attrs = {"ec_techniques": {"ec_technique"}}
+    `ECMeasurement` comes with an `ECPlotter` which either plots `potential` and
+    `current` against time (`ec_meas.plot_measurement()`) or plots `current` against
+    `potential (`ec_meas.plot_vs_potential()`).
 
-    def __init__(self, ec_technique=None, **kwargs):
-        super().__init__(**kwargs)
+    It turns out that keeping track of current, potential, and selector when combining
+    datasets is enough of a job to fill a class. Thus, the more exciting
+    electrochemistry-related functionality should be implemented in inheriting classes
+    such as `CyclicVoltammagram`.
+    """
+
+    extra_column_attrs = {
+        "ec_meaurements": {
+            "ec_technique",
+            "RE_vs_RHE",
+            "R_Ohm",
+            "raw_potential_names",
+            "A_el",
+            "raw_current_names",
+        }
+    }
+
+    def __init__(
+        self,
+        name,
+        *,
+        technique=None,
```

Comment:
Isn't it inheriting some of these kwargs already from the __init__ of the Measurement class? Is there a reason to have them here again?

Reply:
I wrote it all here to be explicit, so that `help(ECMeasurement)` will give all the info. I don't know how else to ensure that. Perhaps @KennethNielsen does.